docs/internal-documentation/maintenance/ReleaseProcess-create-test-distribution.md

```diff
@@ -87,23 +87,28 @@ MANIFEST.SKIP, i.e. no missing or extra files:
 
 > This section is relevant for Zonemaster-GUI only.
 
-The requirements are nodejs and npm. There are available from the [Node.js]
-official website. Minimal version of Nodejs is 10.0 but install the last LTS
-version available. It was tested on Ubuntu 18.04.
+For this you need a [build environment for nodjs], on which you create
```

Comment:
I think this should be `nodejs` (and maybe `Nodejs`)

Reply:
Good catch. And that was not the only place were I had it wrong. They write i `Node.js` and I guess there is no reason not to follow that. I'll correct.